content/_index.en.html

```diff
@@ -20,7 +24,18 @@ <h1>Simplifying Crypto</h1>
   <section class="welcome" id="welcome">
     <h1>Welcome To NavCoin</h1>
     <div class="grid-container ninety-vw-container">
-      <img src="/images/homepage/Apple-Devices2@2x.png" alt="NavCoin Wallets" class="left-img">
+      <!-- <img src="/images/homepage/Apple-Devices2@2x.png" alt="NavCoin Wallets" class="left-img"> -->
+      <div class="container-eco">
+        <a class="video-eco" title="Test title" href="https://www.youtube.com/watch?v=kIC-AflIY2Q">
```

Comment:
If we have the final video ready upload it to youtube as a private video and put the url in here.
WHen we are ready to release the video we can merge this PR and set the video to public on youtube.

We also need to fix the `title="Test title"`